PoGo.NecroBot.Logic/Tasks/UseNearbyPokestopsTask.cs

```diff
@@ -215,18 +224,17 @@ private static bool SearchThresholdExceeds(ISession session)
                 }
             }
         }
-        public static async Task<FortData> GetNextPokeStop(ISession session)
-        {
 
+        private static async Task<FortData> GetNextPokeStop(ISession session)
+        {
             var priorityTarget = await SetMoveToTargetTask.GetTarget(session);
             if (priorityTarget != null) return priorityTarget;
 
-
             if (session.Forts == null ||
                 session.Forts.Count == 0 ||
                 session.Forts.Count(p => p.CooldownCompleteTimestampMs < DateTime.UtcNow.ToUnixTime()) == 0)
             {
-                //non pokestop . should we init or return nul?
```

Comment:
you remove my comment, that line for todo a better logic. temporary leave it like that,